packages/common/src/hooks/purchaseContent/types.ts

```diff
@@ -7,6 +7,17 @@ export type PayExtraAmountPresetValues = {
   [PayExtraPreset.HIGH]: number
 }
 
+export enum PurchaseMethod {
```

Comment:
looks like there are multiple impls of this. you can leave as-is, but after you merge we can consolidate between, this, ray's version, and mine. lol

Reply:
ooo could you share a code pointer plz? i did not see the others